docs/bcfusion/validators/creation.md

```diff
@@ -1,2 +1,137 @@
-This page will guide validators to create new native BSC validators.
-Please be patient for updates.
\ No newline at end of file
+import Validator1 from '../../assets/bcfusion/create-validator1.png';
+import Validator2 from '../../assets/bcfusion/create-validator2.png';
+import Validator3 from '../../assets/bcfusion/create-validator3.png';
+import Validator4 from '../../assets/bcfusion/create-validator4.png';
+
+# Validator Creation
+
+If you are migrating old validators created on
+the BNB Beacon Chain, please refer to [this document](migrations.md) for more information.
+
+You can new the BNB staking dApp to create your validators.
+
+BNB Staking dApp on testnet: https://testnet-staking.bnbchain.org/en/bnb-staking
+
+BNB Staking dapp on mainnet: To be updated.
+
+After opening the dapp, you can click the `Become a Validator` button to create a new validator.
+
+The following information will be required in the `Create Validator` page.
+
+## Basic Info
+
+<img src={Validator1} width="400"/>
+
+- **Validator Name**: The name of your validator.
```

Comment:
Please describe the limitation rule of the name, like the length, character..

Reply:
Added